Sources/python/SimEx/Utilities/wpg_to_opmd.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import numpy
+import math
+import h5py
+import os
+
+from SimEx.Utilities import OpenPMDTools as opmd
+
+def _convertToOPMD(h5_in):
+    """ Take native wpg output and rewrite in openpmd conformant way. """
+
+    # Load the generated h5 file.
+    #wpg_outputs = [os.path.join( self.output_path, f) for f in os.listdir( self.output_path ) ]
+    wpg_outputs = [h5_in]
+    wpg_outputs = [f for f in wpg_outputs if f.split('.')[-1] == 'h5']
+    print wpg_outputs
```

Comment:
not a python 3 compatible `print` statement :)

also: let us add a prefix like `print("processing files: ", wpg_outputs)`